pkg/loader/executor.go

```diff
@@ -102,17 +109,41 @@ func (tx *tx) commit() error {
 	return errors.Trace(err)
 }
 
+func (e *executor) updateMark(status int, channel string, tx *tx) error {
```

Comment:
s/status/initValue  ?    or just remove this param, set it zero or one in updateMark.